### DISCUSSION TITLE
Folder fpdb_profiles

### DISCUSSION THREAD

**Discussion by sf-87** [jejellyroll-fr/fpdb-3#51] [development]
Hi @jejellyroll-fr
I found the "fpdb_profiles" folder into the user directory, but searching for it in the code did not produce any results, so I think that one or more python plugin might generate it. Inside there's some files that seems logs about usage and timing. Do you have any clue? I don't like to have files/folders that I don't want or I'm not aware of, so if we find what is creating them maybe we could change it.

**Comment by jejellyroll-fr** [comment]
Hello Samuele
Don't worry, they are generated by the cprofile library to analyse when it is compiled, where the bottlenecks are and which functions consume the most resources. You can disable them if you wish.

**Comment by sf-87** [comment]
How can I disable it then?

**Reply by jejellyroll-fr** [reply]
Just comment :
profiler.enable()

**Reply by sf-87** [reply]
Sorry to bother again but I can't find that code line nowhere

**Reply by jejellyroll-fr** [reply]
Oh. ... Weird !!!
I put profile on fpdb.pyw
Start by searching import cprofile ... You will find it... probably you are not on the last release

**Reply by sf-87** [reply]
Damn me, I have the case sensitive enabled on visual studio search since who-knows-when-and-why so cprofile != cProfile and it didn't find it
